tests/manage/pv_services/test_pvc_assign_pod_node.py

```diff
@@ -158,3 +180,5 @@ def test_rwx_pvc_assign_pod_node(self, interface, pvc_factory, teardown_factory)
         # Check IO from all pods
         for pod_obj in pod_list:
             pod.get_fio_rw_iops(pod_obj)
+
+        self.verify_access_token_notin_odf_pod_logs()
```

Comment:
Please also make sure that the existing test case in polarion is updated with this new step